packages/reactivity/src/scripts/ast-analyzer.py

```
@@ -0,0 +1,380 @@
+
+                analysis.append(
+                    {
+                        "id": block["id"],
+                        "definedVariables": output_variables,
+                        "usedVariables": [],
+                        "importedModules": [],
+                    }
+                )
+            elif block["type"] == "big-number":
+                used_variables = []
+                if "metadata" in block and block["metadata"]:
```

Ruff (0.14.10)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)